src/app/showcase/doc/messages/servicedoc.ts

```diff
@@ -75,8 +75,8 @@ export class MessagesServiceDemo {
 
     addMultiple() {
         this.messageService.addAll([
-            {severity:'success', summary:'Service Message', detail:'Via MessageService'},
-            {severity:'info', summary:'Info Message', detail:'Via MessageService'}
+            { severity: 'success', summary: 'Service Message', detail: 'Via MessageService', closable: true },
```

Comment:
remove this change.  No need to explicitly set closable

Reply:
Agreed, it helps for clarity in my use case, but I agree that it's probably not the most common use case. Nice catch!